test/unit/math/mix/fun/cholesky_factor_constrain_test.cpp

```diff
@@ -3,7 +3,8 @@
 TEST(mathMixMatFun, cholesky_factor_constrain) {
   auto f = [](int M, int N) {
     return [M, N](const auto& x1) {
-      return stan::math::cholesky_factor_constrain(x1, M, N);
+      stan::scalar_type_t<std::decay_t<decltype(x1)>> lp = 0.0;
```

Comment:
[question]
Doesn't `scalar_type_t` do all the decaying for you?  If not, what's this for?

Reply:
Yep, fixed!